fourkitchens/sous-drupal-recipe
# Sous paragraphs recipe

### Description
Installs paragraph modules and creates three paragraph types with the same fields, form and view display, and media that we had in the original Sous distro.

- text
- text_with_media
- image
- video

**Recipe includes the following contrib modules:**
- paragraphs
- paragraphs_collapsible
- paragraphs_ee
- paragraphs_features

### Testing steps:

1. Run through the install process included in the README. You should see this recipe applied after sous_base and sous_admin.
2. Use the provided link to log in. Verify the paragraph types and media are present.
